include/NovelRT/Animation/SpriteAnimatorState.h

```diff
@@ -11,12 +11,22 @@ namespace NovelRT::Animation {
   class SpriteAnimatorState {
   private:
     std::vector<SpriteAnimatorFrame> _frames;
-    bool _shouldLoop;
+    bool _shouldLoop = false; //this is to shut the warning up since we have no ctor
```

Comment:
Don't we need a ctor to explicitly zero init the vectors?

Reply:
MSVC wasn't complaining about those, only the bool. /shrug

Want me to do it in a ctor anyway?

Reply:
Might be worth testing in Release mode to ensure we don't get wonky behavior, but otherwise I think is fine